.circleci/set-up-globals.sh

```diff
@@ -32,7 +32,7 @@ echo 'export BRANCH=$(echo $CIRCLE_BRANCH | grep -v '"'"'^\(master\|[0-9]\+.x\)$
 echo 'export PR_ENV=${BRANCH:+pr-$BRANCH}' >> $BASH_ENV
 echo 'export CIRCLE_ENV=ci-$CIRCLE_BUILD_NUM' >> $BASH_ENV
 # If we are on a pull request
-if [[ $CIRCLE_BRANCH != "master" && -n ${CIRCLE_PULL_REQUEST+x} ]]
+if [[ ${CIRCLE_BRANCH} != "master" && ! -z ${CIRCLE_PULL_REQUEST} ]]
```

Comment:
I thought this line was the problem, but now I think we just have a race condition: https://github.com/pantheon-systems/example-wordpress-composer/pull/34#issuecomment-328122556